libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -453,12 +455,41 @@ public void appendMediaFile(final MediaFile mediaFile, final String mediaUrl, Im
                 // TODO: insert video
                 ToastUtils.showToast(getActivity(), R.string.media_insert_unimplemented);
             } else {
-                // TODO: insert image
-                ToastUtils.showToast(getActivity(), R.string.media_insert_unimplemented);
+                imageLoader.get(mediaUrl, new ImageLoader.ImageListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
```

Comment:
Shouldn't we be doing something here to let the user know the image failed to be inserted? Same thing [here](https://github.com/wordpress-mobile/WordPress-Android/pull/5761/files#diff-ffbdd6cd0e03b7a13fcdb13666c6d8c3R468).